repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py

```diff
@@ -30,3 +31,21 @@ def get_boot_file_paths():
                                       details={'details': 'Did not receive a message about the leapp-provided'
                                                           'kernel and initramfs'})
     return boot_content.kernel_path, boot_content.initram_path
+
+
+def fix_grub_config_error():
+    conf_file = '/etc/default/grub'
+    with open(conf_file, 'r') as f:
+        config = f.read()
+
+    # move misplaced '"' to the end
+    pattern = 'GRUB_CMDLINE_LINUX=.+(?=GRUB)'
```

Comment:
This won't work in case `GRUB_CMDLINE_LINUX` is the last envvar in the file.

Reply:
You can add a new test file for example with this content:
```
GRUB_TERMINAL_OUTPUT="console" GRUB_CMDLINE_LINUX="vconsole.font=latarcyrheb-sun16 vconsole.keymap=us "rhgb quiet
```
Also, it would be great to have tests that try to fix the test files with the syntax error.